src/notifier.js

```diff
@@ -1005,6 +1012,14 @@ function _processPayload(url, accessToken, payload, callback) {
   // go ahead and send it.
   XHR.post(url, accessToken, payload, function xhrCallback(err, resp) {
     if (err) {
+      if (err instanceof ConnectionError) {
+        // We're calling the callback now with the error, disable the callback for future attempts.
+        payloadObject.callback = function () { };
+        setTimeout(function () {
+          directlyEnqueuePayload(payloadObject);
+        }, 1000 * 10);
```

Comment:
Let's move this timeout into constant. I think future iterations can implement the exponential backoff by storing `payloadObject._retryAttempts` and using that to calculate the timeout + know when to drop the error altogether.